src/main/java/jenkins/plugins/gerrit/AbstractGerritSCMSource.java

```diff
@@ -183,16 +182,16 @@ private boolean isOpenChange(String refName, HashSet<Integer> openChanges) {
   }
 
   /** {@inheritDoc} */
-  @NonNull
+  @Nonnull
   @Override
   protected List<Action> retrieveActions(
-      @CheckForNull SCMSourceEvent event, @NonNull TaskListener listener)
+      @CheckForNull SCMSourceEvent event, @Nonnull TaskListener listener)
       throws IOException, InterruptedException {
     return doRetrieve(
         new Retriever<List<Action>>() {
           @Override
           public List<Action> run(
-              GitClient client, String remoteName, Changes.QueryRequest queryRequest)
+              GitClient client, GitSCMSourceContext context, String remoteName, Changes.QueryRequest queryRequest)
```

Comment:
This seems a bit different from the other cleanups. What was this causing in terms of user-facing problems?